src/__tests__/main.test.ts

```diff
@@ -32,7 +32,9 @@ test.beforeEach(async (t) => {
       weight real,
       make varchar,
       model varchar,
-      primary key (id)
+      serial_key varchar,
```

Comment:
a [`serial_number`](https://thebestbikelock.com/bike-serial-number/) is an actual entity with bikes. i have not heard it referred to as a `serial_key`. willing to use `serial_number` or `serial_num`?